ui2/src/pages/listing/New.tsx

```diff
@@ -172,9 +184,21 @@ const NewComponent: React.FC<RouteComponentProps & ServicePageProps<Service>> =
             required
             onChange={(_, change) => setDescription(change.value as string)}
           />
+          <Form.Select
+            className="select"
+            label="Cleared by"
+            placeholder="Select..."
+            required
+            value={clearedBy}
+            options={[
+              createDropdownProp('-- Collateralized Market --', COLLATERALIZED_VALUE),
+              ...clearedMarketServices.map(cms => createDropdownProp(getName(cms.payload.provider), cms.payload.provider)),
+            ]}
+            onChange={(_, change) => setClearedBy(change.value as string)}
+          />
           <Form.Input label="Trading Calendar ID" required readOnly placeholder={calendarId} />
           <div className="submit">
-            <Button type="submit" className="ghost" disabled={!canRequest} content="Submit" />
+            <Button type="submit" className="ghost" disabled={false} content="Submit" />
```

Comment:
possibly left over from debugging - if not no need for `disabled={false} ` (afaik)

Reply:
good catch, thank you